src/Components/test/E2ETest/Tests/ComponentRenderingTest.cs

```diff
@@ -46,6 +46,10 @@ public void BasicTestAppCanBeServed()
         public void CanRenderTextOnlyComponent()
         {
             var appElement = Browser.MountTestComponent<TextOnlyComponent>();
+
+            new WebDriverWait(Browser, TimeSpan.FromSeconds(10)).Until(
```

Comment:
You can do `Driver.Exists(By.XPath("//*[contains(., 'Hello from TextOnlyComponent')]"))` which should do the waiting etc

Reply:
Good call! Updated.